sway/debug_log.c

```diff
@@ -71,6 +71,11 @@ void layout_log(const swayc_t *c, int depth) {
 				layout_log(c->floating->items[i], depth + 1);
 			}
 		}
+	} else if (c->type == C_ROOT && scratchpad->length > 0) {
+		fprintf(stderr, "ScratchPad\n");
```

Comment:
doop

Reply:
Im guessing you want that line removed?
i left it in, so the scratchpad list is more clear. the output for example
```
focus:R      |(0x5555557a9320)(p:(nil)   )(f:0x55555583d4a0)(h:18446744073709551615)Type:root|layout:-    |w:1680|h:1818|x:   0|y:   0|g:  0|vis:t|children: 1|name:root
|focus:X     |(0x55555583d4a0)(p:0x5555557a9320)(f:0x55555583d820)(h: 1)Type:op  |layout:-    |w:1014|h: 746|x:   0|y:   0|g: -1|vis:t|children: 1|name:WLC-1
|-focus:W    |(0x55555583d820)(p:0x55555583d4a0)(f:0x555555fd6430)(h:18446744073709551615)Type:ws  |layout:Vert |w:1014|h: 746|x:   0|y:   0|g: -1|vis:t|children: 2|name:1
|--focus:X   |(0x555555fd4f90)(p:0x55555583d820)(f:(nil)   )(h: 4)Type:view|layout:-    |w:1014|h: 373|x:   0|y:   0|g: -1|vis:t|children: 0|name:Wayland Terminal
|--focus:K   |(0x555555fd6430)(p:0x55555583d820)(f:(nil)   )(h: 2)Type:view|layout:-    |w:1014|h: 373|x:   0|y: 373|g: -1|vis:t|children: 0|name:Wayland Terminal
ScratchPad
focus:X      |(0x5555560e69f0)(p:(nil)   )(f:(nil)   )(h: 3)Type:view|layout:-    |w: 507|h: 559|x: 253|y:  93|g: -1|vis:f|children: 0|name:Wayland Terminal
focus:K      |(0x555555fd6430)(p:0x55555583d820)(f:(nil)   )(h: 2)Type:view|layout:-    |w:1014|h: 373|x:   0|y: 373|g: -1|vis:t|children: 0|name:Wayland Terminal
```

Reply:
Ah. Alright.